app/src/screens/StakingTokens.js

```diff
@@ -0,0 +1,218 @@
+  }, [myStakes, myActiveTokens])
+
+  const inactiveTokens = useMemo(() => {
+    if (!stakeToken.balance || !myActiveTokens) {
+      return
+    }
+    return stakeToken.balance - myActiveTokens
+  }, [stakeToken.balance, myActiveTokens])
+
+  return (
+    <Box heading="Staking tokens" padding={0}>
+      <div
+        css={`
+          border-bottom: 1px solid ${theme.border};
+        `}
+      >
+        <div
+          css={`
+            padding: ${3 * GU}px;
+          `}
+        >
+          <Field
+            label="Your tokens"
+            css={`
+              margin-bottom: 0px;
+            `}
+          >
+            <div
+              css={`
+                ${textStyle('title2')};
+              `}
+            >
+              {`${
+                stakeToken.balance
+                  ? formatTokenAmount(
+                      parseInt(stakeToken.balance),
+                      parseInt(stakeToken.tokenDecimals)
+                    )
+                  : '-'
+              } ${stakeToken.tokenSymbol}`}
+            </div>
+            <div
+              css={`
+                ${textStyle('body4')};
+                color: ${theme.contentSecondary};
+              `}
+            >
+              {stakeToken.balance
+                ? (parseInt(stakeToken.balance) /
+                    parseInt(stakeToken.tokenSupply)) *
+                  100
+                : '-'}
+              % of total tokens
+            </div>
+          </Field>
+        </div>
+      </div>
+      {myStakes && myStakes.length > 0 && (
+        <div
+          css={`
+            border-bottom: 1px solid ${theme.border};
+          `}
+        >
+          <div
+            css={`
+              padding: ${3 * GU}px;
+            `}
+          >
+            <Field
+              label="Your active tokens"
+              css={`
+                margin-bottom: 0px;
+              `}
+            >
+              <div
+                css={`
+                  ${textStyle('title2')};
+                  color: ${theme.infoSurfaceContent};
+                `}
+              >
+                {`${
+                  myActiveTokens
+                    ? formatTokenAmount(
+                        parseInt(myActiveTokens),
+                        parseInt(stakeToken.tokenDecimals)
+                      )
+                    : '-'
+                } ${stakeToken.tokenSymbol}`}
+              </div>
+              <div
+                css={`
+                  ${textStyle('body4')};
+                  color: ${theme.contentSecondary};
+                `}
+              >
+                Total Active Tokens:{' '}
+                {formatTokenAmount(
+                  parseInt(totalActiveTokens),
+                  parseInt(stakeToken.tokenDecimals)
+                )}{' '}
+                {stakeToken.tokenSymbol}
+              </div>
+            </Field>
+
+            <Field
+              label="SUPPORTED PROPOSALS"
```

Comment:
```suggestion
              label="Supported proposals"
```

Reply:
On the dessings are uppercase

Reply:
Ah yeah, it will be uppercase since the `Field` component transforms the labels. Note that in all other `Fields` we are passing them as suggested.